actions/socket-export-sbom/main.go

```diff
@@ -0,0 +1,132 @@
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+	"log"
+	"net/http"
+	"os"
+)
+
+const BaseURL = "https://api.socket.dev/v0"
+
+type Client struct {
+	APIKey     string
+	BaseURL    string
+	HTTPClient *http.Client
+}
+
+func NewClient(apiKey string) *Client {
+	return &Client{
+		APIKey:     apiKey,
+		BaseURL:    "api.socket.dev/v0",
+		HTTPClient: http.DefaultClient,
+	}
+}
+
+type Repo struct {
+	Slug       string
+	LastScanID string `json:"head_full_scan_id"`
+}
+
+// doRequest handles the common HTTP request logic: building URL, creating request,
+// adding auth header, executing request, and reading response body.
+// Returns the response body bytes, status code, and error.
+func (c *Client) doRequest(method, path string) ([]byte, int, error) {
+	URL := fmt.Sprintf("%s%s", BaseURL, path)
+	authHeaderValue := fmt.Sprintf("Bearer %s", c.APIKey)
+	req, err := http.NewRequestWithContext(context.Background(), method, URL, nil)
+	if err != nil {
+		return nil, 0, err
+	}
+	req.Header.Add("Authorization", authHeaderValue)
+	resp, err := c.HTTPClient.Do(req)
+	if err != nil {
+		return nil, 0, err
+	}
+	defer resp.Body.Close()
+
+	data, err := io.ReadAll(resp.Body)
+	if err != nil {
+		return nil, resp.StatusCode, err
+	}
+	return data, resp.StatusCode, nil
+}
+
+func (client *Client) GetRepoLastScanID(name string) (string, error) {
+	path := fmt.Sprintf("/orgs/grafana/repos/%s", name)
```

Comment:
Same as `ExportSBOM`, I think it would be best if we could parametrize the org.
```suggestion
func (client *Client) GetRepoLastScanID(org, name string) (string, error) {
	path := fmt.Sprintf("/orgs/%s/repos/%s", org, name)
```